.gitignore

```
@@ -1,30 +1,40 @@
 # This file specifies intentionally untracked files that Git should ignore.
 # Files already tracked by Git are not affected.
 # See: https://git-scm.com/docs/gitignore
-.mypy_cache
-.pytest_cache
-.python-version
-__pycache__
-.venv
+## Amazon Web Services ##
+.lambda
 .lambda_other
 .lambda_scikit
-.lambda
-.vs
-htmlcov
 local_testing.py
 
-.vscode
+## Docker ##
+Dockerfile-x
 
-.profile-data
+## Editors ##
+.idea
+.vs
+.vscode
+.vsenv
 
+## Environment ##
 .env
+.venv
+config.ini
 old.env
-src/staticfiles
-created_dummy_data_*.json
+
+## Nodejs ##
 node_modules
-/.vsenv
 
+## Other ##
 *.mmdb
 .geoipupdate.lock
-config.ini
-Dockerfile-x
+.profile-data
+created_dummy_data_*.json
+src/staticfiles
+
+## Python ##
+.mypy_cache
+.pytest_cache
+.python-version
+__pycache__
```

These aligns with [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library/blob/develop/.gitignore). Please remove the `.coverage` file as part of this change.

```suggestion
__pycache__
.coverage
.mypy_cache
.pytest_cache
.python-version
```

---

@xlyk Please remove the `.coverage` file from the repository.

---

resolved here https://github.com/cisagov/con-pca-api/pull/632/commits/f7d54bdda05578142a53c8259a5d3766cf653ea4